pvanalytics/__init__.py

```diff
@@ -1,6 +1,4 @@
 from pvanalytics import features   # noqa: F401
-from pvanalytics import filtering  # noqa: F401
-from pvanalytics import fitting    # noqa: F401
```

Comment:
I think the coverage failure can be ignored; it's just complaining that this deletion lowered the count of covered lines.